### DISCUSSION TITLE
Doesn't using chakra ui breaks the NextJS concept?

### DISCUSSION THREAD

**Discussion by Ali-Aref** [chakra-ui/chakra-ui#7393] [General]
I am quite new NextJS, so please correct me if I am misunderstanding. while using chakra with nextjs we have to put `"use client"` and make the component a client component to work with chakra. **so every piece of element that we are showing using chakara is a client component**. then what is the use of server components here. It become something like pure reactjs. what is the difference ?

**Comment by keithburgie** [comment]
You can wrap Chakra components inside server components: 
https://beta.nextjs.org/docs/rendering/server-and-client-components#third-party-packages
